test/ParseableInterface/module-cache-init.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %empty-directory(%t)
```

Comment:
Nitpick: I'm starting to think we should have a subdirectory for cache tests, like Driver/Dependencies/. They're written in a very different style from the rest of the parseable interface tests.

Reply:
Ok. I'll move them in a followup PR?

Reply:
done